mailpoet/lib/Subscribers/InactiveSubscribersController.php

```diff
@@ -100,6 +101,7 @@ private function deactivateSubscribers(Carbon $thresholdDate, int $batchSize, ?i
     // Select subscribers who received at least a number of emails after threshold date and subscribed before that
     $startId = (int)$startId;
     $endId = $startId + $batchSize;
+    $lifetimeEmailsThreshold = self::LIFETIME_EMAILS_THRESHOLD;
     $inactiveSubscriberIdsTmpTable = 'inactive_subscriber_ids';
     $connection->executeQuery("
       CREATE TEMPORARY TABLE IF NOT EXISTS {$inactiveSubscriberIdsTmpTable}
```

Comment:
I tried the simplification we talked about [here](https://github.com/mailpoet/mailpoet/pull/4004#discussion_r822392647) but the integration tests fail
I think it is better to create a separate issue if we want to investigate the simplification.